### PR TITLE
Don't make program name "qTox" traduisible

### PR DESCRIPTION
**Pull request by TheNain38**

### CODE REVIEW

File: src/loginscreen.ui [hunk 1]
```diff
@@ -23,10 +23,10 @@
    </size>
   </property>
   <property name="windowTitle">
-   <string>qTox</string>
+   <string notr="true">qTox</string>
   </property>
   <property name="windowIcon">
-   <iconset resource="../../res.qrc">
+   <iconset resource="../res.qrc">
```
Comment (tux3) [inline src/loginscreen.ui:29]: What are those path changes for?

**Comment by TheNain38** [comment]
Qt creator atomatically added these

File: src/loginscreen.ui (hunk 1 above)
Comment (TheNain38) [inline src/loginscreen.ui:29]: Qt creator atomatically added these

Comment (TheNain38) [inline src/loginscreen.ui:29]: @tux3 But these new paths are right?

Comment (tux3) [inline src/loginscreen.ui:29]: Yep, the new paths are right actually.

**Pull request merged**